### DISCUSSION TITLE
Autoscrolling when clicking a tab

### DISCUSSION THREAD

**Discussion by al3xandru** [zellij-org/zellij#1424] [General]
I have noticed that in the rare occasions I'm using the mouse to click on a zellij tab, the action registers but then it also triggers an automatic scroll up that must be stopped by hitting a key.

While I understand that the lesson here is "don't use the mouse" 🥲 , I find this behavior very unexpected. It also requires extra care to get back to the point of cursor (my tendency is to hit return until I realize I might have left a command dangling and I don't really need it to run).

Is this considered expected behavior?

System details: zellij 0.28.1 (rollbacked from 0.29.1 due to the resizing bug) on macOS 12.3.1 with the Terminal.app

**Comment by imsnif** [comment]
Hey, this is a bug. It was fixed iirc in 0.29.1

**Comment by al3xandru** [comment]
@imsnif I've just noticed this behavior again in 0.30.0.
